tests/testcases/multiple-entries-mjs/main-a.d.ts

```diff
@@ -0,0 +1 @@
+export { A } from "./common";
```

Comment:
```suggestion
export { A } from "./common.mjs";
```

I wonder what happens when you also include the extension in the source here.

Reply:
Ah okay, lets move on with this solution then.

Reply:
I try `export { A } from "./common.js";` and it all works well in both rollup-plugin-typescript2 and rollup-plugin-dts.